starters/next-react-query-tailwind/src/components/Greeting/Greeting.tsx

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from "react-query";
+
+export function Greeting() {
```

Comment:
I know these are relatively simple components, but the idea of a `Component.data` and `Component.view` separation is pretty core to this kits implementation and should be respected. The idea is that views are simple and don't require much testing where data components have logic that typically require lots of testing. I'd like you to split these components back out into separate files for consistency with our intent of this kit.